docs-source/content/samples/simple/domains/soa-domain/_index.md

```diff
@@ -0,0 +1,605 @@
+---
+title: "SOA domain"
+date: 2019-04-18T07:32:31-05:00
+weight: 3
+description: "Sample for creating a SOA Suite domain home on an existing PV or
+PVC, and the domain resource YAML file for deploying the generated SOA domain."
+---
+
+
+The sample scripts demonstrate the creation of a SOA Suite domain home on an
+existing Kubernetes persistent volume (PV) and persistent volume claim (PVC). The scripts
+also generate the domain YAML file, which can then be used to start the Kubernetes
+artifacts of the corresponding domain. Optionally, the scripts start up the domain,
+and SOA server pods and services based on the domain type (for example, soa, osb, soaosb, soaess and soaessosb).
```

Comment:
Please remove below statement:
"Optionally, the scripts start up the domain,
and SOA server pods and services based on the domain type (for example, soa, osb, soaosb, soaess and soaessosb)."